plugins/modules/backupunit.py

```diff
@@ -293,10 +301,17 @@ def get_sdk_config(module, sdk):
     password = module.params.get('password')
     api_url = module.params.get('api_url')
 
-    conf = {
-        'username': username,
-        'password': password,
-    }
+    if token is not None and token:
```

Comment:
seems like this might introduce some unwanted behaivour:

make that change, then do `export IONOS_TOKEN=`

in this case token is still not None but is empty, and username & password should be used instead